src/vaadin-avatar.html

```diff
@@ -47,8 +56,26 @@
 
         static get properties() {
           return {
+            /**
+             * The path to the image
+             */
+            img: {
+              type: String,
+              reflectToAttribute: true
+            },
+
+            /** @private */
+            __imgVisible: {
+              type: Boolean,
+              computed: '__computeImgVisible(img)',
+              value: false
```

Comment:
Is this something that can be removed? IIRC we shouldn't use `value` and `computed` together.

Reply:
Updated. `__updateVisibility` and `__imgChanged` for handling visibility are added.
`__updateVisibility` will be used with `name` and `abbr` in later PRs